BREAKING CHANGE: rename ReadyToUse to SnapshotCaptured (#195)
cybozu-go/mantle: docs/design.md

status:
  conditions:

>     # The corresponding backup data is ready to use if `status` is "True"

Reviewer: ```suggestion
    # The corresponding backup data has been captured if `status` is "True".
```
